server-spi/src/main/java/org/keycloak/models/ClientModel.java

```diff
@@ -199,6 +200,15 @@
     boolean isServiceAccountsEnabled();
     void setServiceAccountsEnabled(boolean serviceAccountsEnabled);
 
+    default boolean isOAuth2DeviceAuthorizationGrantEnabled() {
```

Comment:
I think it will be better to have this inside the OIDCAdvancedConfigWrapper class instead of ClientModel class. OIDCAdvancedConfigWrapper already contains all of OIDC related properties.

Reply:
I fixed.

Reply:
I see there is still constant OAUTH2_DEVICE_AUTHORIZATION_GRANT_ENABLED on the ClientModel. Can this be removed from ClientModel?

Actually, I figured that there is separate class OAuth2DeviceConfig added for all the properties related to device-grant. So probably it will be even better to remove the configuration from OIDCAdvancedConfigWrapper and move the configuration option into the OAuth2DeviceConfig class. That will be the cleanest approach IMO. WDYT?